release/NativeEditPlugin/Plugins/iOS/EditBox_iOS.m

```diff
@@ -389,8 +390,25 @@ -(void) create:(JsonObject*)json
     else
     {
         uiFont = [UIFont systemFontOfSize:fontSize];
-    }    
-    
+    }
+
+    NSString* contentType;
```

Comment:
Maybe use `UITextContentType` as the type and explicitly initialise to nil for clarity?